src/v/redpanda/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ v_cc_library(
     ${swag_files}
     cluster_config_schema_util.cc
     admin_server.cc
+    admin_debug.cc
```

Comment:
nit: can we just move these into a folder. i.e. `admin/debug.cc`

Reply:
done